README.md

```diff
@@ -273,6 +274,12 @@ db-migrate supports the concept of environments. For example, you might have a d
     "driver": "sqlite3",
     "filename": ":memory:"
   },
+	
+  "int": {
```

Comment:
Yeah I see what you mean, thats for real confusing. But meant is the environment itself, where pg was a defined environment in that case.

Going to improve this...

There are a lot of thinks that are not perfect yet and I have not found them all myself since I'm reworking on this project ;)

So don't hesitate to point to everything confusing.